MhLabs.APIGatewayLambdaProxy/LambdaEntryPointBase.cs

```diff
@@ -129,7 +129,7 @@ private void InitHooks()
             _httpClient = _httpClient ?? new HttpClient(new AwsSignedHttpMessageHandler { InnerHandler = new HttpClientHandler() }) { BaseAddress = Env.Get("ApiBaseUrl").ToUri() };
         }
 
-        [LambdaSerializer(typeof(Amazon.Lambda.Serialization.Json.JsonSerializer))]
+        [LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
```

Comment:
If we change to version 6 - which we should as this upgrade is a breaking change - I guess we can update serialization, but we should verify that it works with existing smoke tests and code pipelines as there has been issues with this in the past.